src/librustdoc/html/static/source-script.js

```diff
@@ -153,7 +154,7 @@ function createSourceSidebar() {
 
 var lineNumbersRegex = /^#?(\d+)(?:-(\d+))?$/;
 
-function highlightSourceLines(match, ev) {
+function highlightSourceLines(match, fromMouseClick) {
```

Comment:
Nit: I like to name function params for _what they do_ rather than _why they are set_. So I'd like to call this `scrollTo` (and reverse the sense).